README.md

```diff
@@ -1071,11 +1071,13 @@ UIAlertController.show(title: "Title", message: "Message", cancelTitle: "Custom
 Access to your device information:
 
 ``` swift
-print(UIDevice.idForVendor()) // 104C9F7F-7403-4B3E-B6A2-C222C82074FF
+print(UIDevice.idForVendor) // 104C9F7F-7403-4B3E-B6A2-C222C82074FF
 print(UIDevice.systemName()) // iPhone OS
 print(UIDevice.systemVersion()) // 9.0
```

Comment:
Why didn't you also make the `systemName` and `systemVersion` computed properties?